templates/people.html

```diff
@@ -18,82 +19,38 @@ <h2 class="text-pink-500 font-normal text-xl leading-snug md:text-4xl lg:lg:text
-                                                    </figure>
-                                                </a>
-                                            {% endif %}
-                                            {% if person.linkedin %}
-                                                <a href="{{ person.linkedin }}">
-                                                    <figure class="rounded-full border-2 border-gray-100 opacity-70 hover:opacity-100 hover:border-gray-200">
-                                                        <img class="w-6 h-6 md:w-7 md:h-7"
-                                                             src="/images/png/linkedin.png" width=32
-                                                             alt="linkedin">
-                                                    </figure>
-                                                </a>
-                                            {% endif %}
-                                            {% if person.goodreads %}
-                                                <a href="{{ person.goodreads }}" target="_blank">
-                                                    <figure class="rounded-full border-2 border-gray-100 opacity-70 hover:opacity-100 hover:border-gray-200">
-                                                        <img class="w-6 h-6 md:w-7 md:h-7"
-                                                            src="/images/png/goodreads.png"
-                                                            alt="goodreads">
-                                                    </figure>
-                                                </a>
-                                            {% endif %}
-                                        </div>
-                                    </div>
-                                </div>
-                                <div class="pt-10 sm:pt-0 sm:pl-10 md:pl-18 lg:pl-38 xl:pl-8">
-                                    <div class="text-black text-base leading-relaxed md:text-xl lg:text-2xl md:leading-snug font-light md:pb-7">
-                                        {{ person.bio | safe }}
-                                    </div>
-                                </div>
-                            </div>
-                        </div>
-                    </div>
-                {% endfor %}
-            </div>
+    {#
+        what is going on here?
+        In the preprocessor, we are using the variable `tileset` to hold various intervals of
+        people data. We loop over those and generate a set of tiles. We then insert a non-person
+        element (like an image or a text tile), before resetting the tileset to take a few more
+        people objects and looping and generating those again. Rinse and repeat.
+    #}
+
+    <section id="people-page-grid">
+
+    {% set data = load_data(path="content/data/people.toml", format="toml") %}
```

Comment:
Let's spike on this and try to remove the TOML file

Reply:
Should I wait for your PR and then rework mine accordingly? Removing TOML is a pretty small change, as long as reading your markdown data is possible with `load_data()` or something similar